docs/content.zh/docs/dev/configuration/overview.md

```diff
@@ -36,39 +36,44 @@ specific language governing permissions and limitations
 under the License.
 -->
 
+<a name="project-configuration"></a>
```

Comment:
```suggestion
<a name="project-configuration"></a>

```

Reply:
It's recommended that There should be an empty line between tag and the section title.